main.js

```diff
@@ -9,30 +9,34 @@ function getQuote(){
 }
 
 function createTweet(input){
+    // console.log(input.typeof, input.length)
     var data = input[0];
 
-    var quoteText = $(data.content).text().trim();
-    var quoteAuthor = data.title;
-
-    if (!quoteAuthor.length) {
-        quoteAuthor = "Unknown author";
-    }
-
-    var tweetText = "Quote of the day - " + quoteText + "Author: " + quoteAuthor;
-
-    if (tweetText.length > 140){
-        getQuote();
-    } else {
-        var tweet = tweetLink + encodeURIComponent(tweetText);
-        $(".quote").text(quoteText);
-        $(".author").text("Author: " + quoteAuthor);
-        $(".tweet").attr("href", tweet);
+    if (Array.isArray(input)){
```

Comment:
you can write just `if (!input.length) {return;}` at the begging of function ;)

Reply:
ok, so Array.isArray() is fine, but shows true even if it is an empty list, and we should have at least 1 element